src/components/Modal/Contributors/styles.ts

```diff
@@ -5,9 +5,14 @@ export const ContributorContainer = styled.div`
 `;
 
 export const ContributorImage = styled.img`
+  width: 100%;
+`;
+
+export const ContributorImageContainer = styled.div`
```

Comment:
I think it would make more sense to put the container of the item on top of it on the order. WDT?

Reply:
I agree.